src/common/enclave_interface_types.h

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+enum CreateNodeStatus
+{
+  OK = 0,
+  InternalError = 1,
+  NodeAlreadyCreated = 2,
+  VersionMismatch = 3,
+  ConsensusNotAllowed = 4,
+  TooManyThreads = 5,
+  MemoryNotInEnclave = 6,
```

Comment:
I think I can guess what this is, but I suspect it and the other values could do with comments.